spring-integration-http/src/test/java/org/springframework/integration/http/management/IntegrationGraphControllerParserTests-context.xml

```diff
@@ -12,12 +12,6 @@
 
 	<mvc:annotation-driven />
 
-	<mvc:cors>
-		<mvc:mapping path="/foo"
-					 allowed-origins="http://foo.bar.com"
-					 allowed-methods="GET" />
-	</mvc:cors>
-
-	<graph-controller path="/foo" />
+	<graph-controller path="/foo" allowed-origins="http://foo.bar.com" />
```

Comment:
Fine with me.
Please, proceed that way.

And, eh..., we should finally come up with the auto-configuration for Boot on the matter...